src/SprykerEco/Yves/Payone/Form/BancontactOnlineTransferSubForm.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Yves\Payone\Form;
+
+use Generated\Shared\Transfer\PaymentTransfer;
+use SprykerEco\Shared\Payone\PayoneApiConstants;
+use Symfony\Component\Form\Extension\Core\Type\HiddenType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+class BancontactOnlineTransferSubForm extends OnlineTransferSubForm
+{
+    const PAYMENT_METHOD = 'bancontact_online_transfer';
```

Comment:
PHP7 visibility modifiers

Reply:
Whole module wrote in the same style. I think it's more relevant for new integration.

Reply:
Applicable for all new code.